tamr_unify_client/models/project/mastering.py

```diff
@@ -76,4 +76,18 @@ def published_clusters(self):
         alias = self.api_path + "/publishedClusters"
         return Dataset(self.client, None, alias)
 
+    def record_clusters_with_data(self):
+        """Record clusters with data.
+
+        Call :func:`~tamr_unify_client.models.dataset.resource.Dataset.refresh` from
+        this dataset to generate clusters with data.
+
+        :returns: Record clusters with its corresponding data (i.e. relative_id, name, version).
+        :rtype: :class:`~tamr_unify_client.models.dataset.resource.Dataset`
```

Comment:
maybe add something in the docstrings about how this is a workaround because the API is broken?